README.md

```diff
@@ -44,6 +53,8 @@ f62cebbc2e43        herdbook_main              "/bin/sh -c /entrypo…"   3 hour
 
 ```
 
+If you 
+
```

Comment:
Some collaborators who are new to the project may have trouble understanding the nuances of this paragraph.